e2e_playwright/st_components_v1_test.py

```diff
@@ -31,6 +31,10 @@ def test_components_iframe_rendering(
     # does not use a valid URL.
     assert_snapshot(elements.nth(0), name="st_components-html")
 
+    # Emulate dark theme OS setting:
+    themed_app.emulate_media(color_scheme="dark")
+    assert_snapshot(elements.nth(0), name="st_components-html")
```

Comment:
Should I be seeing a snapshot somewhere?

Reply:
It is testing against the same snapshot as above since emulating the OS color scheme as dark should not cause the snapshot to change its look. Doing this before would have caused an white background during the testing with the `dark` theme.

Reply:
Ah. I missed that. Thanks!